tools/buildsys/src/manifest.rs

```diff
@@ -405,34 +476,44 @@ impl SupportedArch {
 }
 
 #[derive(Deserialize, Debug, PartialEq, Eq, Hash)]
-#[serde(rename_all = "kebab-case")]
-pub(crate) enum GrubFeature {
-    SetPrivateVar,
+#[serde(try_from = "String")]
+pub enum ImageFeature {
+    GrubSetPrivateVar,
```

Comment:
Oh, Ok, I missed this. So the image features *are* being constrained during the parsing of the Cargo.toml